lsif-java/src/main/scala/com/sourcegraph/lsif_java/commands/SnapshotLsifCommand.scala

```diff
@@ -119,6 +120,31 @@ object SnapshotLsifCommand {
           .setSymbol(symbol)
           .build()
         doc.addOccurrences(occ)
+
+        if (isDefinition) {
+          val hover =
+            (
+              for {
+                resultSetId <- lsif.next.get(o.getId)
+                hoverId <- lsif.hoverEdges.get(resultSetId)
+                hover <- lsif.hoverVertexes.get(hoverId)
+              } yield hover
+            ).get
+              .getContentsList
+              .asScala
+              .find { h =>
+                h.getLanguage != Language.UNKNOWN_LANGUAGE.toString.toLowerCase
+              }
+              .get
+              .getValue
+          val symInfo = SymbolInformation
+            .newBuilder()
+            // we cheese it a bit here, as this is less work than trying to reconstruct
+            // a Signature from the pretty-printed Signature, with accompanying logic
+            // to fallback to display_name in SemanticdbPrinters.scala
+            .setDisplayName(hover).setSymbol(symbol).build()
```

Comment:
Can we use `Documentation` instead?

Reply:
Its weird because many other android types _do_ resolve. :shrug: